lib/thesis/view.ex

```diff
@@ -200,7 +208,7 @@ defmodule Thesis.View do
   end
 
   defp editable?(conn) do
-    Application.get_env(:thesis, :authorization).page_is_editable?(conn)
+    !!conn.assigns[:thesis_editable]
```

Comment:
Instead of asking the host application whether a given page is editable over and over, we just cache it in the `conn` assigns.